### PR TITLE
[TIMOB-24941] Android: Fix /modules packaging on Android.

### PR DESCRIPTION
**Pull request by ypbnv**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-24941

**Optional Description:**
Scons 'package' creates an archive without 'modules' folder. 

Changed the regular expression to work with platform specific path separators.

### CODE REVIEW

**Review by garymathews — APPROVED** [review]
CR: **PASS**
FT: **PASS**

File: build/android.js [hunk 1]
```diff
@@ -129,7 +129,8 @@ Android.prototype.package = function (packager, next) {
 		},
 		// Copy over module resources
 		function (cb) {
-			fs.copy(DIST_ANDROID, ANDROID_MODULES, { filter: /\/android(\/titanium\-(.+)?\.(jar|res\.zip|respackage))?$/ }, cb);
+			filterRegExp = new RegExp('\\' + path.sep  + 'android(\\' + path.sep + 'titanium\-(.+)?\.(jar|res\.zip|respackage))?$');
```
Comment (cb1kenobi) [inline build/android.js]: Missing `var`.

**Review by cb1kenobi — CHANGES_REQUESTED** [review]

**Comment by ypbnv** [comment]
@cb1kenobi Updated.

**Review by cb1kenobi — APPROVED** [review]
CR'd and manually tested the regex. Looks good. **APPROVED**.

**Pull request merged**